frontend/src/pages/comparisons/Comparison.tsx

```diff
@@ -15,10 +29,36 @@ const ComparisonPage = () => {
   const searchParams = new URLSearchParams(location.search);
   const series: string = searchParams.get('series') || 'false';
 
-  const { options } = useCurrentPoll();
+  const { options, name: pollName } = useCurrentPoll();
   const tutorialLength = options?.tutorialLength ?? 0;
   const tutorialAlternatives = options?.tutorialAlternatives ?? undefined;
   const tutorialDialogs = options?.tutorialDialogs ?? undefined;
+  const [alreadyMadeComparisons, setAlreadyMadeComparisons] = useState<
+    Array<string>
+  >([]);
+
+  let dialogs;
+  if (tutorialDialogs) {
+    dialogs = tutorialDialogs(t);
+  }
```

Comment:
(minor) Can be rewritten as a `const`:
```suggestion
  const dialogs = tutorialDialogs ? tutorialDialogs(t) : undefined;
```